packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -62,6 +62,8 @@ const Addon = styled.div`
 `;
 
 const Content = styled.div<{ dashed: boolean; cursor: Cursor }>`
+    cursor: ${({ cursor }) => cursor};
```

Comment:
I'm not happy about this change but it didn't work properly for some elements like in AddAccountButton

Reply:
then probably the one below is not needed? 🤔

Reply:
Maybe 🫣 but I can't assure it's working like that in all the places, so I used it this way.